app/models/concerns/message_subscriptions.rb

```diff
@@ -47,7 +47,11 @@ def required_staff_permission(klass)
     if ['Offer', 'Item'].include?(klass)
       return 'can_manage_offer_messages'
     end
-    
+
+    if ['OfferResponse'].include?(klass)
```

Comment:
Why not make the permission `can_manage_offer_response_messages` and then you don't need this extra `if` statement, you can rely on the convention below to find it